src/itemdetails.html

```diff
@@ -1,291 +1,254 @@
 <div id="itemDetailPage" data-role="page" class="page libraryPage itemDetailPage noSecondaryNavPage selfBackdropPage" data-backbutton="true">
 
-    <style>
-        @media all and (min-width: 62.5em) {
-
-            .itemBackdrop {
-                display: none;
-            }
-        }
-    </style>
-
-    <div id="itemBackdrop" class="itemBackdrop noBackdrop">
-        <div class="itemBackdropFader">
+    <div class="backdropWrapper hide">
```

Comment:
The changes in this file aren't clear in the diff so I'll test them once you finalize the pull request.